internal/core/bootstrap.go

```diff
@@ -207,6 +203,18 @@ func Bootstrap(config *BootstrapConfig) (exitCode int, result interface{}, err e
 		return 1, nil, err
 	}
 	meta.CliConfig = cliCfg
+	if *cliCfg.Output != cliConfig.DefaultOutput && *cliCfg.Output != "" {
+		outputFlag = *cliCfg.Output
+		printer, err = NewPrinter(&PrinterConfig{
+			OutputFlag: outputFlag,
+			Stdout:     config.Stdout,
+			Stderr:     config.Stderr,
+		})
+		if err != nil {
+			_, _ = fmt.Fprintln(config.Stderr, err)
```

Comment:
```suggestion
			fmt.Fprintln(config.Stderr, err)
```

I don't think it's required with `fmt.Fprintln`